_data/projects/letterboxd-for-games.yml

```diff
@@ -0,0 +1,16 @@
+name: letterboxd-for-games
+desc: Social video game discovery and gaming journal.
+site: https://github.com/danielgrijalva/letterboxd-for-games
+tags:
+# Note: these are tags categorizing the project, not issue labels.
+- react
+- reactjs
+- javascript
+- python
+- django
+- django-rest-framework
+- video games
+- gaming
+upforgrabs:
+  name: up for grabs :zap:
```

Comment:
@danielgrijalva Your build is failing and I guess it is due to ":" used in this line, can you please check by removing it.

Reply:
If this is the issue and you want to use this label name only then later we can ignore the build error.

Reply:
hi @ritwik12, that's correct, I want to use that label name 😅

Reply:
Let me check for other errors.